repos/system_upgrade/el7toel8/actors/sssdfacts/tests/unit_test.py

```diff
@@ -9,7 +10,12 @@
 
 def get_config(content):
     config = configparser.ConfigParser()
-    config.readfp(StringIO(textwrap.dedent(content)))
+    if hasattr(config, 'read_file'):
```

Comment:
Hmm why are you solving config.read_file\config.readfp trouble each time differently (in comparison with systemfacts/libraries/systemfacts.py solution)?
Another question - if there are 2+ occurrences already, maybe we should have a helper utility function for that? And have 1 linter exception there and not smeared across codebase?

Reply:
You are right, I changed that before the new years eve, today talking with Michal, we decide to check the python version instead of using `hasattr`.. I change my code, but I miss that piece.. :(